File: test07/devowel_file.pl
#!/usr/bin/perl -w

$file = $ARGV[0];

open($f,$file) or die;

for $i(<$f>){
   $i =~ s/[aeiouAEIOU]//g;
   push @a,$i;
}

open($f,'>',$file);

for $i(@a){
   print $f $i;
}

File: test07/devowel_file.py
#!/usr/bin/python3

import sys
import re

lines = []
f = open(sys.argv[1],'r')

for i in f:
   lines.append(i)

f.close()

f = open(sys.argv[1],'w')

for i in lines:
   i = re.sub('[aeiouAEIOU]',"",i)
   f.write(i)

f.close()
#replaced = ('a','e','i','o','u')

#for i in lines:
#   i.replace(replaced,"")

#f = open(sys.argv[1],'w')
#f.write(lines)
#f.close()

